package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "array.prototype.find": "^1.0.0",
+    "noop2": "2.0.0",
```

Comment:
Unrelated to this pull request, but this has a fixed version while the other modules are using `^`.